client/taskenv/env.go

```diff
@@ -939,6 +961,18 @@ func buildUpstreamsEnv(envMap map[string]string, upstreams []structs.ConsulUpstr
 	}
 }
 
+// addNetNamespacePort builds NOMAD_NET_NAMESPACE_{IP,PORT,ADDR}_{port_label}
```

Comment:
As written, I think it's `NOMAD_ALLOC_{INTERFACE,IP,PORT,ADDR}_{port_label}`, but maybe `NOMAD_NET_NAMESPACE_*` would be better?